bundles/CoreBundle/src/Migrations/Version20230424084415.php

```diff
@@ -37,17 +37,19 @@ public function up(Schema $schema): void
 
             foreach ($editables as $editable) {
                 $unserialized = unserialize($editable['data']);
-                if(array_key_exists('attributes', $unserialized)) {
-                    unset($unserialized['attributes']);
-
-                    $editable['data'] = serialize($unserialized);
-
-                    Db\Helper::upsert(
-                        $db,
-                        'documents_editables',
-                        $editable,
-                        $primaryKey
-                    );
+                if(is_array($unserialized)) {
+                    if (array_key_exists('attributes', $unserialized)) {
+                        unset($unserialized['attributes']);
+
+                        $editable['data'] = serialize($unserialized);
+
+                        Db\Helper::upsert(
+                            $db,
+                            'documents_editables',
+                            $editable,
+                            $primaryKey
+                        );
+                    }
                 }
```

Comment:
Why did you nesting the ifs?
```suggestion
                if (is_array($unserialized) && array_key_exists('attributes', $unserialized)) {
                    unset($unserialized['attributes']);
    
                    $editable['data'] = serialize($unserialized);
    
                    Db\Helper::upsert(
                        $db,
                        'documents_editables',
                        $editable,
                        $primaryKey
                    );
                }
```

Reply:
Heat of the moment I guess. We discovered this during update and acted quickly. Thanks!